python/lsst/ap/association/diaForcedSource.py

```diff
@@ -129,7 +129,7 @@ def __init__(self, **kwargs):
         self.makeSubtask("forcedMeasurement",
                          refSchema=afwTable.SourceTable.makeMinimalSchema())
 
-    def run(self, dia_objects, expIdBits, exposure, diffim, apdb):
+    def run(self, dia_objects, expIdBits, exposure, diffim):
         """Measure forced sources on the direct and different images,
         calibrate, and store them in the Apdb.
```

Comment:
Does this docstring need to be updated? `run` no longer does anything with the apdb.